man/make.conf.5

```diff
@@ -652,9 +652,10 @@ terminal to view parallel-fetch progress.
 .TP
 .B parallel\-install
 Use finer\-grained locks when installing packages, allowing for greater
-parallelism. Note that \fIparallel\-install\fR currently has no effect
-unless \fImerge\-wait\fR is disabled. For additional parallelism,
-disable \fIebuild\-locks\fR.
+parallelism. For additional parallelism disable \fIebuild\-locks\fR.
+Also disable \fImerge\-wait\fR for additional parallelism if desired,
+but this is not necessary because \fIparallel\-install\fR cooperates
```

Comment:
Maybe tweak the phrasing here? I think users get confused by the different options and how to achieve maximum parallelism (and the risks of each).

Reply:
Yeah, changed it to:
> Also disable merge-wait for additional parallelism if desired, but that increases the possibility of random build failures. When parallel-install is used together with merge-wait, parallel installation occurs in batches when there are no builds running.